src/DataStructures/Tensor/TypeAliases.hpp

```diff
@@ -285,6 +285,11 @@ using IJJ = Tensor<DataType, tmpl::integral_list<std::int32_t, 2, 1, 1>,
                               SpatialIndex<SpatialDim, UpLo::Up, Fr>,
                               SpatialIndex<SpatialDim, UpLo::Up, Fr>>>;
 template <typename DataType, size_t SpatialDim, typename Fr = Frame::Inertial>
+using iij = Tensor<DataType, tmpl::integral_list<std::int32_t, 2, 2, 1>,
+                   index_list<SpatialIndex<SpatialDim, UpLo::Lo, Fr>,
+                              SpatialIndex<SpatialDim, UpLo::Lo, Fr>,
+                              SpatialIndex<SpatialDim, UpLo::Lo, Fr>>>;
+template <typename DataType, size_t SpatialDim, typename Fr = Frame::Inertial>
```

Comment:
I was using these in an earlier version of the new test helpers for KerrSchild's spin magnitude. Why not leave them, because they might be helpful for others down the road?

Reply:
They're easy to add if/when they are needed, so I personally would favor not adding unused code.

Reply:
If that's the consensus, I'll remove them, and I agree against avoiding unused code in general. But I think this is an exceptional case where it makes sense to keep them. 

Personally, I find it mildly annoying when I'm pulled away from the task at hand to go create a new tensor alias that I want. If I knew how to do it, I would want tensor aliases for any index to just work. So why not keep aliases that do come up in GR once someone bothers to write them, rather than making someone else have to re-add them down the road?